.github/workflows/Build_and_publish_package.yml

```diff
@@ -0,0 +1,33 @@
+name: Build and Deploy Packages
+
+on: [push]
+
+env:
+  # Customize the CMake build type here (Release, Debug, RelWithDebInfo, etc.)
+  BUILD_TYPE: RelWithDebInfo
+jobs:
+  build:
+    # The CMake configure and build commands are platform agnostic and should work equally
+    # well on Windows or Mac.  You can convert this to a matrix build if you need
+    # cross-platform coverage.
+    # See: https://docs.github.com/en/actions/configuring-and-managing-workflows/configuring-a-workflow#configuring-a-build-matrix
+    runs-on: windows-latest
+
+    steps:
+    - uses: actions/checkout@v2
+    
+    - name: Update git submodule
+      # ensure that the libgit2 linkage is correct
+      run: |
+        git submodule init
+        git submodule update
+
+    - name: Build & Deploy Package
+      # Build project using powershell script
+      run: |
+        powershell .\build.libgit2.ps1
```

Comment:
We could probably split this into two steps doing something like:

```
- name: Build LibGit2
   shell: Powershell
   run: .\build.libgit2.ps1
```

If this doesn't work, you can simply specify `shell: Powershell` for this step and then you don't need to explicitly call `powershell`.

Reply:
Split workflow into two steps ;)
Step 1 Build project
Step 2 Publish package